test/System/MemoryInputStream.cpp

```diff
@@ -0,0 +1,33 @@
+#include <SFML/System/MemoryInputStream.hpp>
+#include <string_view>
+#include <string>
+#include <sstream>
+
+#include <doctest.h>
+
+TEST_CASE("sf::MemoryInputStream class - [system]")
+{
+    SUBCASE("Empty stream")
+    {
+        sf::MemoryInputStream mis;
+
+        CHECK(mis.read(nullptr, 0) == -1);
+        CHECK(mis.seek(0) == -1);
+        CHECK(mis.tell() == -1);
+        CHECK(mis.getSize() == -1);
+    }
+
+    SUBCASE("Open memory stream")
+    {
+        const std::string memoryContents = "hello world";
```

Comment:
```suggestion
        using std::literals::string_view_literals;
        constexpr auto memoryContents = "hello world"sv;
```

Reply:
It's possible to make this change but you still require `<sstream>` to build for me locally so we'd only be dropping 1 include `<string>`